doc/nrf/ug_matter_overview_architecture_integration.rst

```diff
@@ -0,0 +1,70 @@
+.. _ug_matter_overview_architecture_integration:
+
+Matter integration in the |NCS|
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Matter is included in the |NCS| as one of the submodule repositories managed with the :ref:`zephyr:west` tool, using a dedicated fork ``nrfconnect/sdk-connectedhomeip``.
+That is, the code used for the |NCS| and Matter integration is stored in the Matter repository (nRF Connect platform) and is compiled when building one of the available :ref:`matter_samples`.
+Both instances depend on each other, but their development is independent to ensure that they both support the latest stable version of one another.
+The fork is maintained and verified as a part of the |NCS| release process.
+
+Matter is located on the top application layer of the integration model, looking from the networking point of view.
+The |NCS| and Zephyr provide the Bluetooth® LE and Thread stacks, which must be integrated with the Matter stack using a special intermediate layer.
+The |NCS|'s Multiprotocol Service Layer (MPSL) driver allows running Bluetooth LE and Thread concurrently on the same radio chip.
```

Comment:
```suggestion
In case of Matter over Thread, the |NCS|'s Multiprotocol Service Layer (MPSL) driver allows running Bluetooth LE and Thread concurrently on the same radio chip.
```